src/sources.js

```diff
@@ -323,6 +325,16 @@ class S3Source extends Source {
     }
     return object.Body;
   }
+  parseOverviewMarkdown(overviewMarkdown) {
+    const frontMatter = yamlFront.loadFront(overviewMarkdown);
+    if (!frontMatter.title || !frontMatter.byline) {
+      throw new Error("Incorrectly formatted frontmatter in overview file");
+    }
+    // handle files with CRLF endings (windows)
+    const content = frontMatter.__content.replace(/\r\n/g, "\n");
+
+    return [frontMatter.title, frontMatter.byline, content];
```

Comment:
Can we also include the following fields in the frontmatter (precise naming can be changed if desired):
* `displayAvailableDatasets` (bool)
* `displayAvailableNarratives` (bool)

And update the customised splash page accordingly.